main/main.cpp

```diff
@@ -1115,7 +1115,6 @@ Error Main::setup(const char *execpath, int argc, char *argv[], bool p_second_ph
 		goto error;
 	}
 #endif
-
```

Comment:
This line and the one removed above are not related to this change and IMO useful for readability, so I'd prefer if they were not removed in this PR.